### DISCUSSION TITLE
[Study schedule] 11/4 9PM(CST)

### DISCUSSION THREAD

**Discussion by swiri021** [OpenKBC/multiple_sclerosis_proj#93] [Study]
@kicheolkim @lacuss Hi guys, I invited you guys to this study, and just in case, let me know if you don't have an invitation from me. Thank you!

**Comment by swiri021** [comment]
## Topic:
1. Code review for AWS module(batch module)
2. Current data analysis status
3. Discussion

**Comment by swiri021** [comment]
@lacuss @kicheolkim This is reminder!, see you on time!

**Comment by swiri021** [comment]
* AWS resource: https://youtu.be/T4aAWrGHmxQ

**Comment by kicheolkim** [comment]
R code for DESeq2
https://github.com/kicheolkim/RNAseq_FullTx/tree/master/R
